### ISSUE TITLE
Guava 32.1 - capability already exists

### ISSUE THREAD

**Issue by kmoens** [gradlex-org/java-ecosystem-capabilities#54]
When we use Guava 32.0, we now get an error that the capability already exists, because Guava now also publishes Gradle metadata, and the capability is part of that one already I believe.

Simple reproduction:
```
plugins {
    id 'java'
    id 'java-library'
    id 'org.gradlex.java-ecosystem-capabilities' version '1.3'
}

repositories {
    mavenCentral()
}

dependencies {
    implementation 'org.assertj:assertj-guava:1.3.1'
}
```

Gives:
```
 ./gradlew dependencyInsight --configuration compileClasspath --dependency guava

> Task :dependencyInsight
com.google.guava:guava:[15.0,) FAILED
   Failures:
      - Could not resolve com.google.guava:guava:[15.0,).
          - Could not resolve com.google.guava:guava:32.1.0-jre.
              - There was an error while evaluating a component metadata rule for com.google.guava:guava:32.1.0-jre.
                  - Cannot add capability com.google.collections:google-collections with version 32.1.0 because it's already defined with version 32.1.0-jre

com.google.guava:guava:[15.0,) FAILED
\--- org.assertj:assertj-guava:1.3.1
     \--- compileClasspath

org.assertj:assertj-guava:1.3.1
  Variant compile:
    | Attribute Name                 | Provided | Requested    |
    |--------------------------------|----------|--------------|
    | org.gradle.status              | release  |              |
    | org.gradle.category            | library  | library      |
    | org.gradle.libraryelements     | jar      | classes      |
    | org.gradle.usage               | java-api | java-api     |
    | org.gradle.dependency.bundling |          | external     |
    | org.gradle.jvm.environment     |          | standard-jvm |
    | org.gradle.jvm.version         |          | 8            |

org.assertj:assertj-guava:1.3.1
\--- compileClasspath

A web-based, searchable dependency report is available by adding the --scan option.
```

**Comment by jjohannes** [comment]
Thanks for letting me know. I am aware of the issue already but I did not know that a 32.1 was published. 

Took me a bit by surprise that suddenly Gradle Metadata is published. After waiting for it for almost 4 years. :)

I will do a patch for this plugin later today.

**Comment by jjohannes** [comment]
Unfortunately, there is also an issue with the published metadata. I'll wait until that is resolved before doing the update here.

https://github.com/google/guava/issues/6612

**Comment by cpovirk** [comment]
Sorry for both the extremely slow publishing of the metadata and the error in the release. Thanks to you both for reporting it.

**Comment by jjohannes** [comment]
Fixed in `1.3.1`
